Src/Sarif.PatternMatcher.Cli/ImportAndAnalyzeOptions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Cli
+{
+    [Verb("import", HelpText = "Export rules metadata to a markdown file.")]
+    internal class ImportAndAnalyzeOptions : AnalyzeOptions
+    {
+        [Option(
+            "temp-folder",
+            HelpText = "A temp folder path to save all files.")]
+        public string TempFolder { get; set; }
```

Comment:
>TempFolder [](start = 22, length = 10)

Delete this argument entirely. Just create a temporary folder using .NET helpers and then make sure it's cleaned up!